ship/uniqueid.go

```diff
@@ -15,3 +16,14 @@ func UniqueID(prefix, appID string) (string, error) {
 
 	return fmt.Sprintf("%s-%0x", prefix, s[:8]), nil
 }
+
+// UniqueID creates ship ID with given prefix and salted by provided protectedID
+// The protectedID is basically the same as `machineid.ProtectedID(appId)` above
+// but provided by the system using this library
+func UniqueIDWithProtectedID(prefix, protectedID string) (string, error) {
```

Comment:
Könntest Du stattdessen eine Funktion reinreichen lassen, so dass evcc die zu verwendende machine-id bestimmen kann und damit den evcc Standard nutzen?

Reply:
Ich reiche den von evcc generierten Wert rein. Siehe hier: https://github.com/evcc-io/evcc/pull/4207/files#diff-57a13a6ebe2afaf22c517ea8970440f3c1a972298fcb3ff91a87341ec6c2fc41

Reply:
Ich würde vorschlagen den Fallback rauszunehmen. Die id muss von evcc kommen; das stellt sicher dass sie da in der Config auch persistiert ist.

Reply:
Alles klar. Ist erledigt.